assignment.js

```diff
@@ -57,13 +64,21 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
+
 console.log('Problem 2:')
 
 // Add your code below this line
+let numArray = [9, 18, 62, 3, 17]
+  let sum = 0
```

Comment:
@jimernest90 be careful with your indentation, line 76 - 80 should be indented back 1 space back to the left; also  

Current code: 
```
let numArray = [9, 18, 62, 3, 17]
  let sum = 0
  numArray.forEach(function(numArray) {
    sum += numArray
  })
  console.log(sum / numArray.length)
```
vs: 

Code optimized for readability:
```
let numArray = [9, 18, 62, 3, 17]
let sum = 0

numArray.forEach(function(numArray) {
    sum += numArray
})

console.log(sum / numArray.length)
```